Neos.Cache/composer.json

```diff
@@ -13,7 +13,7 @@
         "neos/utility-opcodecache": "*"
     },
     "require-dev": {
-        "mikey179/vfsstream": "~1.6",
+        "mikey179/vfsstream": "^1.6.1",
```

Comment:
Is this a conscious change (here and elsewhere?) If so, why?

Reply:
Yes, due to the lowest-dependency test runs. Flow tests won't work with 1.6.0 due to the missing bugfix for `"uninitialized string offset"` (which you reported to vfsstreams btw :))
Lowest dependency tests still don't pass completely, but one issue less at least

Reply:
Got it, thanks!